practice/src/css/layouts/header.css

```diff
@@ -0,0 +1,166 @@
+.header-section {
+  padding: 16px 32px;
+}
+  
+.header-logo {
+  display: flex;
+  align-items: center;
+  gap: 4px;
+}
+
+.header-name {
+  text-decoration: none;
+  font-family: var(--secondary-font);
+  font-weight: var(--font-weight-regular);
+  line-height: 20px;
+  color: var(--secondary-text-color);
+  font-size: var(--font-size-xsmall);
+}
+  
+.header-nav {
+  display: grid;
+  grid-template-columns: 1fr auto;
+  align-items: center;
+  gap: 10px;
+}
+  
+label {
+  cursor: pointer;
+}
+  
+#header-dropdown {
```

Comment:
Don't use ID to style, you should use class instead of  id

https://google.github.io/styleguide/htmlcssguide.html#ID_Selectors